plugins/SentryLogger/SentryLogger.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2019 Ultimaker B.V.
+# Cura is released under the terms of the LGPLv3 or higher.
+
+from UM.Logger import LogOutput
+from typing import Set
+from sentry_sdk import add_breadcrumb
+from typing import Optional
+import os
+
+home_dir = os.path.expanduser("~")
+
+class SentryLogger(LogOutput):
```

Comment:
Would be good to add some documentation here as to what this class (or module) does.